pkg/proc/test/support.go

```diff
@@ -81,6 +81,13 @@ const (
 
 // BuildFixture will compile the fixture 'name' using the provided build flags.
 func BuildFixture(name string, flags BuildFlags) Fixture {
+	return BuildFixtureWithFlags(name, flags, "")
+}
+
+// BuildFixtureWithFlags will compile the fixture 'name' using the provided
+// a la carte build flags as well as additional freeform build flags specified as
+// a string to be passed as-is to the 'go' command.
+func BuildFixtureWithFlags(name string, flags BuildFlags, moreflags string) Fixture {
```

Comment:
Do you plan to use this in the future?

Reply:
I am reverting these changes. I tried to do something with build fixtures for testing, but then came up with a better way, so this is no longer needed.